util/csvresponse.py

```diff
@@ -18,11 +18,13 @@
 INT_PRESSURE_NAME = app.config.get('INT_PRESSURE_NAME')
 
 # default parameters to request
-DEFAULT_PARAMETERS = ['time', 'deployment', 'lat', 'lon']
+DEFAULT_PARAMETERS = ['time', 'deployment', 'lat', 'lon', 'm_gps_lat', 'm_gps_lon']
 
 # Glider GPS based lat/lon strings - for filtering
 GLIDER_GPS_BASED_LAT = 'glider_gps_position-m_gps_lat'
 GLIDER_GPS_BASED_LON = 'glider_gps_position-m_gps_lon'
+GLIDER_BASED_LAT = 'glider_gps_position-m_lat'
```

Comment:
should we rename these something that indicates they are dead-reckoned values? `GLIDER_DR_LAT`?

Reply:
That makes sense. I've refactored GLIDER_BASED_LAT, GLIDER_BASED_LON to GLIDER_DR_LAT, GLIDER_DR_LON.